DSharpPlus.Commands/CommandExecutor.cs

```diff
@@ -54,37 +51,25 @@ private static async ValueTask WorkerAsync(CommandContext context)
 
         if (checks.Count != 0)
         {
-            ContextCheckAttribute check = null!;
-            try
+            List<Exception> failedChecks = [];
+
+            // Reverse foreach so we execute the top-most command's checks first.
+            for (int i = checks.Count - 1; i >= 0; i--)
             {
-                // Reverse foreach so we execute the top-most command's checks first.
-                for (int i = checks.Count - 1; i >= 0; i--)
+                if (!await checks[i].ExecuteCheckAsync(context))
```

Comment:
User implemented checks are executed with the expectation to throw. With the removal of the try/catch statement, how will the behavior change?

Reply:
that'll have to be updated once non-converter attributes are a thing